superglue/lib/utils/url.ts

```diff
@@ -1,79 +1,79 @@
 import parse from 'url-parse'
 
-export function pathQuery(url) {
+export function pathQuery(url: string): string {
```

Comment:
This file contains a few naming changes, but functionally equivalent.

Reply:
Feedback not related to this PR: is there a reason you're using `url-parse` instead of the native Node/browser `URL` and `URLSearchParams` functionality? I feel like those could replace most of the functions in this file.

Reply:
That's because Superglue is older than it looks. The first commit on this project was in 2016, and i had been working on it a few year before that. Back then URL wasn't consistent across browsers yet, and url-parse helped to normalize it. I think its time to remove it, but there's a slight difference in the  API. Its on my todo list!